ch09/WWTravelClubDBCosmos/CosmosDBClientSample/Program.cs

```diff
@@ -36,48 +34,45 @@ public static async Task Main()
             }
         }
 
-        public async Task CreateCosmosDB()
+        public static async Task CreateCosmosDB()
         {
-            using (var cosmosClient = new CosmosClient(endpoint, key))
-            {
-                Database database = await cosmosClient.CreateDatabaseIfNotExistsAsync(databaseId);
-                ContainerProperties cp = new ContainerProperties(containerId, "/DestinationName");
-                Container container = await database.CreateContainerIfNotExistsAsync(cp);
-                await AddItemsToContainerAsync(container);
-            }
+            using var cosmosClient = new CosmosClient(endpoint, key);
+            Database database = await cosmosClient.CreateDatabaseIfNotExistsAsync(databaseId);
+            ContainerProperties cp = new ContainerProperties(containerId, "/DestinationName");
+            Container container = await database.CreateContainerIfNotExistsAsync(cp);
+            await AddItemsToContainerAsync(container);
         }
 
 
-        private async Task AddItemsToContainerAsync(Container container)
+        private static async Task AddItemsToContainerAsync(Container container)
         {
-            Destination destinationToAdd = new Destination()
+            var destinationToAdd = new Destination
             {
                 Id = "1",
                 Country = "Brazil",
                 DestinationName = "São Paulo",
                 Description = "The biggest city in Brazil",
-                Packages = new Package[]
+                Packages = new[]
                 {
-                new Package()
-                {
-                    Id = "1",
-                    Description = "Visit Paulista Avenue",
-                    DuratioInDays = 3,
-                    Price = 5000,
-                }
+                    new Package
+                    {
+                        Id = "1",
+                        Description = "Visit Paulista Avenue",
+                        DurationInDays = 3,
+                        Price = 5000,
+                    }
                 }
             };
 
 
             try
             {
-                ItemResponse<Destination> destination = await container.ReadItemAsync<Destination>(destinationToAdd.Id, new PartitionKey(destinationToAdd.DestinationName));
+                await container.ReadItemAsync<Destination>(destinationToAdd.Id, new PartitionKey(destinationToAdd.DestinationName));
             }
             catch (CosmosException ex) when (ex.StatusCode == System.Net.HttpStatusCode.NotFound)
             {
                 // Create an item in the container representing the Andersen family. Note we provide the value of the partition key for this item, which is "Andersen"
```

Comment:
This comment also needs updating, because the sample works with destinations and not families. I'll leave that for you to change as you wish. You might want to make an additional commit to this PR accordingly, before merging.